samples/README.md

```diff
@@ -0,0 +1,11 @@
+# KFP Tekton Samples
+
+Below are the list of samples that are currently running end to end taking the compiled Tekton yaml and deploying on a Tekton cluster directly. 
+If you are interested more in the larger list of pipelines samples we are testing for whether they can be 'compiled to Tekton' format, please [look at the corresponding status page](/sdk//python/tests/README.md)
```

Comment:
remove the extra slash. (/sdk/python/tests/README.md)

Reply:
The extra slash won't affect the link, but probably is nice to just have single slash to avoid confusion.